nodejs16.x/response-streaming/{{cookiecutter.project_name}}/src/index.js

```diff
@@ -0,0 +1,31 @@
+exports.handler = awslambda.streamifyResponse(
+    async (event, responseStream, context) => {
+        const httpResponseMetadata = {
+            statusCode: 200,
+            headers: {
+                "Content-Type": "text/html",
+                "X-Custom-Header": "Example-Custom-Header"
+            }
+        };
+
+        responseStream = awslambda.HttpResponseStream.from(responseStream, httpResponseMetadata);
+
+        responseStream.write("<html>");
```

Comment:
Shouldn't we use .pipeline() here? That's what we recommend.

Reply:
I added a comment pointing to the documentation instead for people to use pipeline for more complex examples. The problem with using `pipeline()` for this sending simple text is that 
`responseStream.write("<html>");`
becomes
`await pipeline(Readable.from(Buffer.from("<html>")), responseStream);`

so every line that looked simple becomes kind of a mess to accomplish the same. In the future we could add another example that uses pipeline to actually return something streamable in the first place.